doc/source/release-notes/Percona-Server-8.0.25-15.1.rst

```diff
@@ -0,0 +1,47 @@
+.. _8.0.25-15.1:
+
+================================================================================
+*Percona Server for MySQL* 8.0.25-15.1
+================================================================================
+
+:Date: June 16, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.25-15.1
+includes all the features and bug fixes available in
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
```

Comment:
It also contains all changes available in MySQL 8.0.24! These should be included as well.

Reply:
Added.